data/glsk-file/src/main/java/com/farao_community/farao/commons/data/generatorloadshiftkeys/GlskException.java

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
This exception do not give extra information. You may use FaraoException instead ?

Reply:
ok